src/events/message/index.ts

```diff
@@ -19,11 +20,34 @@ export default class MessageEvent extends Event {
 	public async listener(message: Message): Promise<void> {
 		const categories = process.env.CATEGORIES!.split(",");
 
+		if (!(message.channel instanceof GuildChannel) || message.author.bot) {
+			return;
+		}
+
 		if (
-			!(message.channel instanceof GuildChannel) ||
-			message.author.bot ||
-			!categories.includes(message.channel.parentID!)
+			message.content.trim().toLowerCase() === `<@!${this.client.user!.id}> ping` ||
+			message.content.trim().toLowerCase() === `<@${this.client.user!.id}> ping`
 		) {
+			const pingMessage = await message.channel.send("Ping ?").catch(noop);
+
+			if (!pingMessage) {
+				return;
+			}
+
+			const binHealth = await got(`${ORIGIN_URL}/health`).text().catch(noop);
+
+			const embed = new MessageEmbed()
+				.setColor(binHealth === "alive" ? 0x2ab533 : 0xf33030)
```

Comment:
```suggestion
				.setColor(binHealth ? 0x2ab533 : 0xf33030)
```

Reply:
C'est plus explicite, et on peut tout à fait imaginer un scénario où d'autres codes apparaissent (et comme ça on est sûr d'obtenir "alive")

Reply:
non y aura pas d'autres codes, et puis là à moins d'être con c'est assez explicite que si on y arrive jusque là c'est que le bin n'est pas down puisqu'il a su renvoyer une réponse

Reply:
"Y'aura pas d'autres codes" ça on n'en est pas sûr

Reply:
si si, tu veux qu'il dise quoi ? "Online but not yet ready"... ?
Au pire quand y en aura d'autres (jamais) on changera.